pkg/cache/multi_namespace_cache.go

```diff
@@ -99,10 +112,24 @@ func (c *multiNamespaceCache) GetInformerForKind(ctx context.Context, gvk schema
 		}
 		informers[ns] = informer
 	}
+	clusterCacheInf, err := c.clusterCache.GetInformerForKind(ctx, gvk)
+	if err != nil {
+		return nil, err
+	}
+	informers[globalCache] = clusterCacheInf
```

Comment:
we could ideally create a separate informer in [`multiNamespaceInformer`](https://github.com/kubernetes-sigs/controller-runtime/blob/0d84ce43073cefb813563dd7586c943fa2c97783/pkg/cache/multi_namespace_cache.go#L184) for cluster scoped cache. But that's not needed for now, since the namespace is not specifically used in any of the methods.